scripts/train_lookup_lemmatizer.py

```diff
@@ -0,0 +1,43 @@
+from pathlib import Path
+
+import typer
+
+from tqdm import tqdm
+from conllu import parse_incr
+
+from spacy.lookups import Lookups, Table
+
+app = typer.Typer()
+
+
+@app.command()
+def main(input_file: str, output_path: Path, min_occurrences: int = 1):
```

Comment:
1 seems to be a reasonable choice, but I would be curious how does the model perform with values like 3,5,10. (My guess is that much worse.)

Reply:
Yes, it got worse. I tested on the `hu_core_news_lg` model with the full pipeline. 
Here are the results:
1: 97.36%
3: 96.97%
5: 96.94%
10: 96.95%